programs/ssl/ssl_client2.c

```diff
@@ -1494,7 +1599,7 @@ int main( int argc, char *argv[] )
         else if( strcmp( p, "tickets" ) == 0 )
         {
             opt.tickets = atoi( q );
-            if( opt.tickets < 0 || opt.tickets > 2 )
+            if( opt.tickets < 0 || opt.tickets > 1 )
```

Comment:
Why was this change made?

Reply:
The values defined for tickets are 0 or 1.